packages/engine/lib/orchestrator/src/manifest.rs

```diff
@@ -39,10 +39,11 @@ pub struct Manifest {
     pub behaviors: Vec<SharedBehavior>,
     /// A list of all datasets in the project.
     pub datasets: Vec<SharedDataset>,
-    /// JSON string describing the [`Globals`](hash_engine::config::Globals) object.
+    /// JSON string describing the [`Globals`](hash_engine_lib::config::Globals) object.
     pub globals_json: Option<String>,
     /// JSON string describing the analysis that's calculated by the
-    /// [analysis output package](hash_engine::simulation::package::output::packages::analysis).
+    /// [analysis output
+    /// package](hash_engine_lib::simulation::package::output::packages::analysis).
```

Comment:
I think rustdoc won't interpret it as a link. 
```suggestion
    /// [analysis output package].
    /// 
    /// [analysis output package]: hash_engine_lib::simulation::package::output::packages::analysis
```